tests/foreman/ui/test_operatingsystem.py

```diff
@@ -232,27 +231,27 @@ def test_negative_create_os_with_same_name_and_version(self):
                                  (common_locators['haserror']))
 
     @run_only_on('sat')
+    @tier1
     def test_positive_delete(self):
         """@Test: Delete an existing OS
 
         @Feature: OS - Positive Delete
 
-        @Assert: OS is deleted
-
+        @Assert: OS is deleted successfully
         """
         os_name = entities.OperatingSystem().create().name
         with Session(self.browser):
             self.operatingsys.delete(os_name)
 
     @run_only_on('sat')
-    def test_update_os_basic_params(self):
+    @tier1
+    def test_positive_update_random_param(self):
```

Comment:
Not sure if `random_param` is better version, as we're updating OS name, major_version, minor_version, os_family and arch here, no randomness here. Btw, how about just `test_positive_update`?